README.md

```diff
@@ -61,7 +59,7 @@ You should now be able to use riscv64-unknown-elf-gcc and its cousins.
 
 ### Installation (Linux)
 
-To build the Linux cross-compiler, pick an install path (that is writeable.)
+To build the Linux cross-compiler, pick an install path (that is writeable).
```

Comment:
this changes may not related with the title.

Reply:
Thanks.
I updated the title, which might be more appropriate.